multinet/api/tests/test_upload_csv.py

```diff
@@ -0,0 +1,11 @@
+import pathlib
+
+import pytest
+
+data_dir = pathlib.Path(__file__).parent / 'data'
+
+
+@pytest.mark.django_db
+def test_upload_valid_csv(authenticated_api_client):
+    # TODO: Implement once able
```

Comment:
File an issue to track this.

Reply:
https://github.com/multinet-app/multinet-api/issues/29